### PR TITLE
Complete HRMC-RTS-Small-Area PMD4 Migration 

### PR DESCRIPTION
**Pull request by canwaf**
Cube class added, family assigned, dataset a bit janky due to catalog/datatype selection; however it works.

### CODE REVIEW

File: datasets/HMRC-RTS-Small-area/main.py [hunk 1]
```diff
@@ -17,15 +18,18 @@
...
-scraper.select_dataset(latest=True)
-scraper
+tabs = {tab.name: tab for tab in scraper.catalog.dataset[0].distribution[1].as_databaker()}
+# tabs = {tab.name: tab for tab in scraper.distribution(latest=True, mediaType=Excel).as_databaker()}
+# excel_distro_list = [x for x in scraper.catalog.dataset[0].distribution if x.mediaType == ExcelOpenXML]
+# excel_distro_list = [x for x in scraper.catalog.dataset[0].distribution if x.mediaType == Excel]
 
-tabs = {tab.name: tab for tab in scraper.distribution(title=lambda t: 'Data Tables' in t).as_databaker()}
-#tabs.keys()
+excel_distro_list = [x for x in scraper.catalog.dataset[0].distribution if x.mediaType == Excel]
```
Comment (Shannon95) [inline datasets/HMRC-RTS-Small-area/main.py:32]: Remove as catolog & distribution has already been selected and excel_distro_list is not used anywhere else in code.

File: datasets/HMRC-RTS-Small-area/main.py [hunk 2]
```diff
@@ -66,19 +70,10 @@ def process_tab(t):
 #Flow has been changed to Flow Direction to differentiate from Migration Flow dimension
 # -
 
-out = Path('out')
-out.mkdir(exist_ok=True)
-table.to_csv(out / 'observations.csv', index = False)
+cubes.add_cube(scraper, table, info['title'])
+scraper.dataset.family = info['families']
+scraper.catalog.dataset[0].family = info['families']
```
Comment (Shannon95) [inline datasets/HMRC-RTS-Small-area/main.py:75]: From looking into the .py files the main runs (T1, T2, T3, T4, T5) and the output files I can see this is a multi-measure dataset; GBP Total &  Count of Businesses therefore will need split out into two cubes and use the cube class in the way to represent this. Here is an example of how it has previously been done for reference https://github.com/GSS-Cogs/family-trade/blob/master/datasets/ONS-Regional-gross-domestic-product-city-regions/main.py#L464 

how the scraper is handled in each separate notebook (T1 - T5) can change scraper = Scraper(seed="info.json")

it'll pick up the url and family information from the info.json automatically and then lines 74 & 75 can be removed.

**Review by Shannon95 — COMMENTED** [review]
There are some changes needed around using the scraper and seed / it is a multi-measure dataset so will need split out. 

The tracer will also need to be implemented for each tab. 

As this is an older transformation where we separately pulled out each tab into different notebooks it might be easier to bring it into one as a lot of the code is repeated.

File: datasets/HMRC-RTS-Small-area/main.py (hunk 1 above)
Comment (canwaf) [reply datasets/HMRC-RTS-Small-area/main.py:32]: Done.

File: datasets/HMRC-RTS-Small-area/main.py (hunk 2 above)
Comment (canwaf) [reply datasets/HMRC-RTS-Small-area/main.py:75]: From a linked data perspective, the output is atomic (i.e. each record only has one measure type + value). I know it's a multiple measure, but it's being represented in the correct way. I'm going to have to read more up on this to see how it's worked in your example next week. _I am **done** for this week._

Comment (canwaf) [reply datasets/HMRC-RTS-Small-area/main.py]: Fixed.

**Comment by canwaf** [comment]
With this merged, a Data Manager is required to update info.json with the measure types. Re-assign me when that review is completed.

**Pull request merged**